decomposition.tex

```diff
@@ -137,21 +137,21 @@
     // addressability
     { &t } -> Same<T*>;
```

Comment:
Oh wait. I don't think you meant to change this here. This is the `Movable` concept from the Palo Alto report. But the larger issue remains.